ballerina-to-openapi/src/main/java/io/ballerina/openapi/service/mapper/parameter/AbstractParameterMapper.java

```diff
@@ -55,8 +59,15 @@ public void setParameter() throws ParameterMapperException {
         parameterList.forEach(operationInventory::setParameter);
     }
 
-    static Object getDefaultValue(DefaultableParameterNode parameterNode) {
-        Node defaultValueExpression = parameterNode.expression();
+    static Object getDefaultValue(DefaultableParameterNode parameterNode, SemanticModel semanticModel) {
+        ExpressionNode defaultValueExpression = (ExpressionNode) parameterNode.expression();
```

Comment:
Is it safe to cast here or do we have to check the type or kind?

Reply:
Yeah, before adding this,  I checked with the lang team and, they informed that it is safe to cast because it returns expression nodes always.